docs/install/tensorflow-install.md

```diff
@@ -22,11 +20,14 @@ version support matrix:
 | Post-5.7.0 | Last three versions at ROCm release. |
 :::
 
-### Installing TensorFlow
+For current TensorFlow ROCm version support, refer to the
+[GitHub repository](https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/blob/develop-upstream/rocm_docs/tensorflow-rocm-release.md).
```

Comment:
@jayfurmanek here is the link update. Do you have a better name for "GitHub repository" for this link?

Reply:
@saadrahim last version of tensorflow v2.13, not is compiled with 7900xtx